### ISSUE TITLE
I am trying to test client server setup

### ISSUE THREAD

**Issue by alexk345** [caprover/caprover#789]
reactjs+ expressjs + mongodb
I cant create user in mongodb. This one click mongodb , how do you create user?  I went to docker instance and it wont allow me to create admin user or any user.

I mean it has mongodb one click app. But i cant create user . I went inside instance and wentinto mongodb instance. I still cant create user. I need to be admin first with " use admin"  database then i need to create other users in other databases. Issue is there is another security check somewhere that allow you to create admin user.
mongodb.conf or something with auth=true , then some thing else...also... so many versions...its just give me headache to get it working.

use admin
db.createUser({	user: "admin",pwd: "pass",roles: [{ role: 'root', db: 'admin' }]})
This is where i stuck

> db.createUser({user: "admin",pwd: "pass",roles: [{ role: 'root', db: 'admin' }]})
uncaught exception: Error: couldn't add user: command createUser requires authentication :
_getErrorWithCode@src/mongo/shell/utils.js:25:13
DB.prototype.createUser@src/mongo/shell/db.js:1343:11
@(shell):1:1

Can some one help solve this issue.

**Comment by githubsaturn** [comment]
> I went inside instance and wentinto mongodb instance. I still cant create user.
How are you connecting to your mongodb shell? When you set up an instance, you set a root username and a password. You first need to connect to the instance via that username and password, then you should be able to interact with the database.

PS: this is really not a CapRover question. I suggest you ask the question on StackOverflow or a MongoDB forum. You have a better chance for finding an answer.

**Comment by alexk345** [comment]
Solved it. Issue is i have to start mongodb  with 
mongo -u root 
it will ask for password. once you provide it works
one of the mistake people make or i did is...not putting use db before any command
use demotest
db.createUser({user: "testuser",pwd: "somepass", roles: [ { role: "readWrite", db: "demotest" } ] })




But i have new problem. 

I did curl -i localhost:8080/users  - it work
but my client calling my server with same http://localhost:8080/users  it hangs
